python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -378,17 +378,16 @@ def test_non_existent_list_versions_latest_only(arctic_library):
 
 def test_delete_version_with_snapshot(arctic_library):
```

Comment:
Do we have VersionStore coverage for the ` iterate_snapshots_if_tombstoned=False` case?

Reply:
The V2 API tests implicitly cover the V1 API doing the right thing of the `kwarg` is provided